R/install_easystats.R

```diff
@@ -40,7 +40,7 @@ install_latest <- function(source = c("development", "cran"),
   source <- match.arg(source, c("development", "cran"))
   pkg <- .packages_on_cran()
 
-  if (all(packages == "all")) {
+  if (packages == "all") {
```

Comment:
packages can be vector length > 1, so this will warn/error.

Reply:
I don't understand this. 

If `packages` can be a character vector, won't this _always_ to evaluate to `FALSE`? 

```r
(packages == "all")
```

Reply:
Oh, I see, I get it.

Reply:
I have changed the code to make it easier to reason about.